CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
 set(MIR_VERSION_MAJOR 2)
 set(MIR_VERSION_MINOR 20)
-set(MIR_VERSION_PATCH 0)
+set(MIR_VERSION_PATCH 2)
```

Comment:
This should now be `2.21.0`?

Reply:
That depends on whether we're rolling a 2.20.2?